### ISSUE TITLE
140279 - Stove Guard

### ISSUE THREAD

**Issue by late4marshmellow** [NamronAS/Namron_Homey#196]
Have issue with keeping the stoveguard connected to my homey. 
Right after pairing the stove guard leaves the zigbee network, or stay  but stops reporting anything. 
as of now after last repair, last report is 6days ago on all 4 statuses. 
Though checking https://tools.developer.homey.app/tools/zigbee
the last report from the stoveguard is 2 hours ago.

**Comment by Taadbv** [comment]
Same here my stove guard 1402791 drops out of zigbee arter few second after tøying to reparere or reinstall. Nameron app dose not recongice the stove guard 1402791, only ad it as a unknown Device.

**Comment by andersthorsen** [comment]
Same here. 140279*

**Comment by zub** [comment]
Indeed, apparently this seems to be an issue. The device decides to partially leave the network after a few seconds. It is usually still possible to turn the guard off via zigbee, but it then disconnects completely.

**Comment by prj84** [comment]
We see that the stove guard is not very fond off being bridged by other devices, so we would like to recomend if possible to make a direct connection between Homey and the device.

You can also test if you have the same issues if you try to use the Fire Fence app on Homey to see if you have the same problems. Please report back and I will set you in contact with our support team of the stove guard developer.

**Comment by late4marshmellow** [comment]
> We see that the stove guard is not very fond off being bridged by other devices, so we would like to recomend if possible to make a direct connection between Homey and the device.
> 
> You can also test if you have the same issues if you try to use the Fire Fence app on Homey to see if you have the same problems. Please report back and I will set you in contact with our support team of the stove guard developer.

the layout of my house wont allow a direct connection to my homey, still this should not be a issue with a Zigbee certified device since a mesh network is the whole idea and upside of this system...

Tried to reset remove and readd the device with the FireFence app, but when pair is a success the device connects itself to the Namron App. i do not want to delete the namron app just to test this since it will destroy my flows on my other 14 Namron devices!

**Comment by prj84** [comment]
Hi! We are in dialog with developer/manufacturer to find a solution to the problem that a minor of our users are experiencing. We will get back as soon as we have any updates regarding this.
